modules/database/src/adapters/sequelize-adapter/utils/schema.ts

```diff
@@ -136,12 +137,20 @@ export async function getTransactionAndParsedQuery(
   } else {
     parsedQuery = query;
   }
-  if (parsedQuery.hasOwnProperty('$set')) {
+  if (method === 'PATCH') {
+    if (!parsedQuery.hasOwnProperty('$set')) {
```

Comment:
if it doesn't have $set you don't need to do anything here.

Reply:
> if it doesn't have $set you don't need to do anything here.

> you need to check if the property is missing not if it is already there. If method is PATCH and parsedQuery has $set, then nothing is needed. If method is PATCH and $set is missing you add it. Also if method is PUT but $set exists then we just do patch. $set shouldn't be added on PUT

Reply:
I got it mixed with mongo. $set doesn't do anything in sequelize so it shouldn't remain in the query. So to re-iterate: if method is PATCH and the query contains $set, you remove it and keep whatever is inside. If $set exists no matter the method we need to do "PATCH".